frontend/public/components/storage-class-form.tsx

```diff
@@ -103,6 +103,44 @@ export class StorageClassForm_ extends React.Component<
   // For 'csi' storage type
   CSIStorageTypes = Object.freeze({
     ...this.getExtensionsStorageClassProvisioners(Provisioner.CSI),
+    'ebs.csi.aws.com': {
+      title: 'AWS CSI',
+      provisioner: 'ebs.csi.aws.com',
+      allowVolumeExpansion: true,
+      parameters: {
+        type: {
+          name: 'Type',
+          values: { io1: 'io1', gp2: 'gp2', sc1: 'sc1', st1: 'st1', standard: 'standard' },
+          hintText: 'Select AWS Type. Default is gp2',
+        },
+        iopsPerGB: {
+          name: 'IOPS Per GiB',
+          hintText: 'I/O operations per second per GiB',
+          validation: (params) => {
+            if (params.iopsPerGB.value !== '' && !params.iopsPerGB.value.match(/^[1-9]\d*$/)) {
+              return 'IOPS per GiB must be a number';
+            }
+            return null;
+          },
+          visible: (params) => _.get(params, 'type.value') === 'io1',
+        },
+        fsType: {
+          name: 'Filesystem Type',
+          hintText: 'Select Filesystem to Be Laid Out. Default is ext4',
```

Comment:
This is odd case, but I see it's the same as below. Should it be "to be laid out"?

Reply:
How about going with this?
`Selected File system type that will be formatted during volume creation. Default Type is ext4`

Reply:
Maybe "Filesystem type to use during volume creation. Default is ext4." Punctuation on both sentences. Filesystem is one word.